src/components/element-editor/basic-details/dropdown-items/dropdown-items-bin-button.tsx

```diff
@@ -0,0 +1,48 @@
+import { PopoverTrigger } from "@lifesg/react-design-system/popover-v2";
+import { StyledBinButton } from "./dropdown-items-child.styles";
+
+interface IProps {
+    onClick: () => void;
+    disabled?: boolean;
+    popoverMessage?: string | React.ReactNode;
```

Comment:
so you don't need to cast it in `PopoverTrigger` 

```suggestion
    popoverMessage?: JSX.Element;
```